apps/zero-runtime-vite-app/src/App.tsx

```diff
@@ -32,10 +33,19 @@ const bounceAnim = keyframes({
   },
 });
 
+declare module '@mui/zero-runtime' {
+  interface ThemeArgs {
+    theme: Theme & {
```

Comment:
@mnajdova I've left the typings for theme to be provided in user land. Let me know if this makes sense.

Reply:
I am still not 100% sure about this - the main reason is that some of the `styled()` functionality depends on a certain Theme structure, like components' variants, some of the structure for things used in the sx prop, the vars object etc. I am fine with extending it - that makes total sense.

Reply:
Again, let's isolate the theme module augmentation in a dedicated file.